types/moment-range/package.json

```diff
@@ -1,5 +1,6 @@
 {
     "dependencies": {
-        "moment": ">=2.14.0"
+        "moment": "2.17.1",
+        "moment-range": "3.0.3"
```

Comment:
Hi.
Yes, although the functionality can be "used" directly it can be also "used" as an extension of the moment library (moment-range acts as an extension of the current moment entity), therefore I add both dependencies.
If you refer to not add the dependencies with a more flexibility, you are right it should be added that way. But this dependencies are only used for the unit testing.
Thanks!